zipkin-cassandra/src/test/scala/com/twitter/zipkin/storage/cassandra/CassandraSpanStoreSpec.scala

```diff
@@ -36,4 +40,23 @@ class CassandraSpanStoreSpec extends SpanStoreSpec {
   override lazy val store = new CassandraSpanStore(new Repository(keyspace, cluster))
 
   override def clear = cluster.connect().execute("DROP KEYSPACE IF EXISTS " + keyspace)
+
+  override def setTimeToLive() {
+    ready(store(Seq(span1)))
+    ready(store.setTimeToLive(span1.traceId, 1234.seconds))
+
+    for( i <- 1 to 100) {
+      // @xxx Repository.storeXXX() methods write asynchronously but don't return a Future so we can't reliably test.
+      //  just wait and loop instead
+      java.lang.Thread.sleep(50)
+      // If a store doesn't use TTLs this should return Duration.Top
```

Comment:
But we know whether the Cassandra store uses TTLs, no? So we could just have the specialized code here?